nf_core/bump_version.py

```diff
@@ -48,17 +49,36 @@ def bump_pipeline_version(pipeline_obj: Pipeline, new_version: str) -> None:
     )
     # multiqc_config.yaml
     multiqc_new_version = "dev" if "dev" in new_version else new_version
+    multiqc_current_version = "dev" if "dev" in current_version else current_version
+    if multiqc_new_version == "dev":
+        update_file_version(
+            Path("assets", "multiqc_config.yml"),
+            pipeline_obj,
+            [
+                (
+                    f"/releases/tag/{current_version}",
+                    f"/tree/dev",
+                )
+            ],
+        )
+    if multiqc_current_version == "dev":
```

Comment:
I think I'm not following this. Why are we updating the file based on the current version instead of the new version?

Reply:
I am assuming one would update from `dev->release` and `release->dev`` so I know how to replace what. but maybe that's actually to simple?